### PR TITLE
Ropes refactor save image link

### PR DESCRIPTION
**Pull request by kkajla12**

### CODE REVIEW

File: test/controllers/images_controller_test.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@ class ImagesControllerTest < ActionDispatch::IntegrationTest
     post images_path('image' => {
       'url' => url
     })
-    assert_response :found
+    assert_redirected_to image_path(id: 1)
```
Comment (bboe) [inline test/controllers/images_controller_test.rb]: What would happen to this assertion of someone where to add/modify the setup function to create a handful of images? How can you make it more robust?

Comment (kkajla12) [inline test/controllers/images_controller_test.rb]: Is this better: c36089e0041b952be5874767023fd78726e70038?

Comment (bboe) [inline test/controllers/images_controller_test.rb]: Yes.

**Comment by bboe** [comment]
👍  please squash the relevant commits.

**Pull request merged**